scripts/cray_runall.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+alps_present=`which alps`
+if [ $? == 0 ]; then
+launcher="aprun -n 1"
+else
+launcher="srun -n 1"
+fi
+echo Running fi_info
+$launcher ./fi_info 2>&1 | tee test.output
+if [ $? != 0 ] ; then
+exit -1
+fi
+count=`grep gni test.output | wc -l`
+if [ $count != 3 ] ; then
+echo "GNI doesn't appear in fi_info about, aborting..." 
+exit -1
+fi
+rm ./test.output
+sleep 1
+echo Running fi_ep_test
+$launcher ./fi_ep_test -f gni
+if [ $? != 0 ] ; then
+exit -1
```

Comment:
If this test or any of the remaining tests fail, what if we just caught the error and continued running the rest of the tests, exiting with -1 if any of them failed.  Could also print out a summary of number of tests failed and succeeded.